manifests/rbac.yaml

```diff
@@ -0,0 +1,55 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cosi-controller-role
+rules:
+- apiGroups: ["objectstorage.k8s.io"]
+  resources: ["bucketrequests", "bucketaccessrequests"]
+  verbs: ["get", "list", "watch"]
```

Comment:
Do we need update?  Doesn't the controller-manager update the BR with references to the B?

Reply:
https://github.com/kubernetes-sigs/container-object-storage-interface-controller/issues/19